Exercise 1 BFile/myFile-rt/src/com/tridiumuniversity/myFile/BFile.java

```diff
@@ -0,0 +1,214 @@
+package com.tridiumuniversity.myFile;
+
+import javax.baja.nre.annotations.NiagaraAction;
+import javax.baja.nre.annotations.NiagaraProperty;
+import javax.baja.nre.annotations.NiagaraType;
+import javax.baja.sys.*;
+
+@NiagaraType
+@NiagaraProperty(
+        name = "fileName",
+        type = "BString",
+        defaultValue = "BString.DEFAULT"
+)
+@NiagaraProperty(
+        name = "path",
+        type = "BString",
+        defaultValue = "BString.DEFAULT"
+)
+@NiagaraProperty(
+        name = "size",
+        type = "BInteger",
+        defaultValue = "BInteger.DEFAULT"
+)
+@NiagaraProperty(
+        name = "createdDate",
+        type = "BAbsTime",
+        defaultValue = "BAbsTime.make()"
+)
+@NiagaraProperty(
+        name = "modifiedDate",
+        type = "BAbsTime",
+        defaultValue = "BAbsTime.NULL"
+)
+
+@NiagaraAction(
+        name = "touch"
+)
+
+@NiagaraAction(
+        name = "print"

```

Comment:
This is a little nitpicky, but I notice that you're indenting 8 spaces in the annotations, but only 2 spaces below. It's probably best to be consistent.